packages/java/tests/spring/native/pom.xml

```diff
@@ -28,6 +28,11 @@
         <dependency>
             <groupId>com.vaadin</groupId>
             <artifactId>vaadin-lumo-theme</artifactId>
+            <version>24.1.10</version>
```

Comment:
Why do you need the version here?

Reply:
It's wrong btw :)

Reply:
All the dependencies could be replaced by the standard dev.hilla/hilla dependencies. See how the other tests were changed